source/retryable-writes/tests/README.rst

```diff
@@ -296,9 +296,28 @@ supported write operations:
   - ``bulkWrite()`` with ``ordered=true`` (no ``UpdateMany`` or ``DeleteMany``)
   - ``bulkWrite()`` with ``ordered=false`` (no ``UpdateMany`` or ``DeleteMany``)
 
+Prose Tests
+===========
+
+The following tests ensure that retryable writes work properly with replica sets
+and sharded clusters.
+
+#. Test that retryable writes raise an exception when using the MMAPv1 storage engine,
```

Comment:
Let's use a hardcoded test number here instead of an autogenerated one. See https://jira.mongodb.org/browse/SPEC-1364 for the rationale.

Reply:
Done